Pipfile

```diff
@@ -15,6 +15,7 @@ pyelftools = "*"
 packaging = "*"
 toml = "*"
 aiohttp = "*"
+semantic-version = "*"
```

Comment:
Is the semantic-version still used? I think we replaced it with packaging. It can be removed from `thoth/python/aiosource.py` (unused) and from requirements.txt.

Reply:
It seems to break the following tests - 
```
=========================== short test summary info ============================
FAILED tests/test_aiosource.py::TestAIOSource::test_get_package_hashes_warehouse
FAILED tests/test_pipfile.py::TestPipfile::test_from_string[Pipfile_test1] - ...
FAILED tests/test_pipfile.py::TestPipfile::test_construct_requirements - Asse...
FAILED tests/test_pipfile.py::TestPipfileLock::test_from_string[Pipfile_test1.lock]
FAILED tests/test_pipfile.py::TestPipfileLock::test_construct_requirements_txt
FAILED tests/test_source.py::TestSource::test_get_package_hashes_warehouse - ...
FAILED tests/test_source.py::TestSource::test_get_package_hashes_server - Typ...
FAILED tests/test_source.py::TestSource::test_get_packages - TypeError: unhas...
FAILED tests/test_source.py::TestSource::test_get_package_versions_warehouse
FAILED tests/test_source.py::TestSource::test_get_package_versions_simple - T...
```

Reply:
Can it be considered as a dev dependency in that case?

Reply:
The test's seem to be failing even after adding as a dev dependency